assignments/prototypes.js

```diff
@@ -59,15 +59,29 @@ GameObject.prototype.createdAt = function(created) {
   * should inherit takeDamage() from CharacterStats
 */
   function  Humanoid(attrs) {
-    dimensions: {
-      length: 2;
-      width: 1;
-      height: 1;
-   }
-   this.team = attrs.team;
-   this.weapons = attrs.weapons;
+    CharacterStats.call(this, attrs);
+    this.createdAt = attrs.createdAt;
+    this.name = attrs.name;
+    this.team = attrs.team;
+    this.weapons = attrs.weapons;
+    this.healthPoints = attrs.healthPoints;
+    this.language = attrs.language;
+
   }
- 
+  Humanoid.prototype = Object.create(CharacterStats.prototype); //this one take the f from the char stats ....inherit
+
+  Humanoid.prototype.destroy = function(){
```

Comment:
This one should be on `GameObject`. Humanoid should inherit instead of having it defined on it's own prototype.